kaldb/pom.xml

```diff
@@ -437,6 +431,10 @@
                     <groupId>org.slf4j</groupId>
                     <artifactId>slf4j-log4j12</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>com.fasterxml.jackson.module</groupId>
+                    <artifactId>jackson-module-scala_2.13</artifactId>
```

Comment:
This was causing us problems attempting to target a newer databind release, and doesn't appear necessary for our use case.

Reply:
Please add this as a comment also, so we don't delete it by accident in future.

Reply:
Done.